src/main/scala/mesosphere/marathon/core/volume/providers/DockerHost.scala

```diff
@@ -0,0 +1,47 @@
+package mesosphere.marathon.core.volume.providers
+
+import com.wix.accord.Validator
+import com.wix.accord.combinators.NilValidator
+import mesosphere.marathon.core.volume._
+import mesosphere.marathon.state._
+import org.apache.mesos.Protos.{ Volume => MesosVolume }
+
+/**
+  * DockerHostVolumeProvider handles Docker volumes that a user would like to mount at
+  * predetermined host and container paths. Docker host volumes are not intended to be used
+  * with "non-local" docker volume drivers. If you want to use a docker volume driver then
+  * use a PersistentVolume instead.
+  */
+protected case object DockerHostVolumeProvider
+    extends ContextUpdateHelper[DockerVolume]
+    with VolumeProvider[DockerVolume] {
+  val name = "docker" // only because we should have a non-empty name
```

Comment:
or maybe make `name` an Option field? but require that all "registered" providers have a name?